test_divide.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from divide import divide, IncorrectInput
+
+
```

Comment:
Really good work here Sandra! I would give this suite a 7/10 as you are missing _one_ kind of test case. It would also be good to see some comments here if you want to make it even clearer what the test is setting out to do 😄

Reply:
I have added some comments and more test cases. Please let me know if there's an important one that I'm missing. Not quite sure if I can think of another one :)